src/Symfony/Component/PropertyInfo/composer.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "symfony/property-info",
+    "type": "library",
+    "description": "Symfony Property Info Component",
+    "keywords": [
+        "property",
+        "type",
+        "PHPDoc",
+        "symfony",
+        "validator",
+        "doctrine"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Kévin Dunglas",
+            "email": "dunglas@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.9"
+    },
+    "require-dev": {
+        "symfony/phpunit-bridge": "~2.7",
+        "symfony/serializer": "~2.7",
+        "phpdocumentor/reflection": "~1.0",
+        "doctrine/annotations": "~1.0"
+    },
+    "conflict": {
+        "phpdocumentor/reflection": "<1.0.7"
```

Comment:
I understand the reason for the conflict block and agree that we should keep it. But still, I'd set the version constraint in `require-dev` to `^1.0.7`. That would be the semantically correct way to express that we require at least version 1.0.7, which is obviously the case.